src/components/interface/AppNavBar.tsx

```diff
@@ -15,16 +17,15 @@ export default function AppNavBar() {
   // Get the config for the current step
   const studyConfig = useStudyConfig();
   const step = useCurrentStep();
-  const currentStepConfig = studyConfig.components[step];
+  const stepConfig = studyConfig.components[step];
 
-  const trialId = useCurrentTrial();
+  const currentConfig = isPartialComponent(stepConfig) && studyConfig.baseComponents ? merge({}, studyConfig.baseComponents?.[stepConfig.baseComponent], stepConfig) as IndividualComponent : stepConfig as IndividualComponent;
```

Comment:
This and some other spots could lead to some issues with the config. I think we need to fail more gracefully. Let's open an issue for that

Reply:
#178